server/model/workSpace/workSpace.model.ts

```diff
@@ -0,0 +1,78 @@
+import { Column, DataSource, Entity, PrimaryColumn, Repository } from "typeorm";
+
+import { useLog } from "../../lib/log";
+
+// ? what is this for?
+const log = useLog({
+  dirname: __dirname,
+  filename: __filename,
+});
+
+@Entity({ name: "workspace", schema: "public" })
+export class Workspace {
+  @PrimaryColumn("uuid")
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column({ name: "user_id" })
```

Comment:
You should add a `workspaceToUser` table for supporting the ability to share and collaborate on a workspace.